### PR TITLE
Email DT fix

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-35103

## Description
A small bug in CSP caused enriched Email data to be added under the wrong key in context.
This is also how the DT is defined in the Email indicator type.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ShahafBenYakir — DISMISSED** [review]
As discussed, this breaks BC, need to asses the damage and decide if:
1. Is it worth the risk, and 2. If it does make the appropriate release notes to alert customers.

File: Packs/Base/ReleaseNotes/1_33_52.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  2.  Cofense Intelligence V2
+  3.  Eclectic IQ Intelligence Center V3
+  4.  Email Hippo
+  5.  IP Quality Score
+  6.  MISP V3
+  7.  Reversing Labs Titanium Cloud V2
+  8.  SEKOIA Intelligence Center
+  9.  Threat Crowd V2
+  10. Threat Zone
+  11. VM Ray
```
Comment (ShirleyDenkberg) [inline Packs/Base/ReleaseNotes/1_33_52.md:18]:
```suggestion
Fixed an issue which caused email enrichers to save results in the wrong context key. `Email` key was changed to 
`Accounts.Email`. This change might affect the following integrations which output Email indicators to context:
- Anomali Threat Stream V3
- Cofense Intelligence V2
- Eclectic IQ Intelligence Center V3
- Email Hippo
- IP Quality Score
- MISP V3
- Reversing Labs Titanium Cloud V2
- SEKOIA Intelligence Center
- Threat Crowd V2
- Threat Zone
- VM Ray
```

File: Packs/MISP/ReleaseNotes/2_1_43.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### MISP v3
+
+Due to a fixed issue in `Base` pack (Version `1_33_52`) MISP will now correctly input email addresses into context under `Accounts.Email` and not under `Email` as it did up until now.
```
Comment (ShirleyDenkberg) [inline Packs/MISP/ReleaseNotes/2_1_43.md:6]:
```suggestion
Fixed an issue in `Base` pack (Version `1_33_52`) so now MISP will correctly input email addresses into context under `Accounts.Email` and not under `Email`.
```

**Comment by ShirleyDenkberg** [comment]
@ShahafBenYakir Doc review completed.

File: Packs/Base/ReleaseNotes/1_34_4.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
+#### Scripts
+
+##### CommonServerPython
+
+Fixed an issue which caused email enrichers to save results in the wrong context key. `Email` key was changed to 
+`Accounts.Email`. This change might affect the following integrations which output Email indicators to context:
```
Comment (ShahafBenYakir) [inline Packs/Base/ReleaseNotes/1_34_4.md:7]:
```suggestion
Fixed an issue which caused email enrichers to save results in the wrong context key. `Email` key was changed to `Accounts.Email`. This change might affect the following integrations which output Email indicators to context:
```

File: Packs/Base/ReleaseNotes/1_34_4.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": "Fixed an issue which caused email enrichers to save results in the wrong context key. `Email` key was changed to \n`Accounts.Email`. This change might affect the following integrations which output Email indicators to context:\n  1.  Anomali Threat Stream V3\n  2.  Cofense Intelligence V2\n  3.  Eclectic IQ Intelligence Center V3\n  4.  Email Hippo\n  5.  IP Quality Score\n  6.  MISP V3\n  7.  Reversing Labs Titanium Cloud V2\n  8.  SEKOIA Intelligence Center\n  9.  Threat Crowd V2\n  10. Threat Zone\n  11. VM Ray"
```
Comment (ShahafBenYakir) [inline Packs/Base/ReleaseNotes/1_34_4.json:3]:
```suggestion
    "breakingChangesNotes": "Fixed an issue which caused email enrichers to save results in the wrong context key. `Email` key was changed to `Accounts.Email`. This change might affect the following integrations which output Email indicators to context:\n  1.  Anomali Threat Stream V3\n  2.  Cofense Intelligence V2\n  3.  Eclectic IQ Intelligence Center V3\n  4.  Email Hippo\n  5.  IP Quality Score\n  6.  MISP V3\n  7.  Reversing Labs Titanium Cloud V2\n  8.  SEKOIA Intelligence Center\n  9.  Threat Crowd V2\n  10. Threat Zone\n  11. VM Ray"
```

File: Packs/MISP/ReleaseNotes/2_1_43.json [hunk 5]
```diff
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": "Due to a fixed issue in `Base` pack (Version `1_33_52`) MISP will now correctly input email addresses into context under `Accounts.Email` and not under `Email` as it did up until now."
```
Comment (ShahafBenYakir) [inline Packs/MISP/ReleaseNotes/2_1_43.json:3]:
```suggestion
    "breakingChangesNotes": "Due to a fixed issue in `Base` pack (Version `1.33.52`) MISP will now correctly input email addresses into context under `Accounts.Email` and not under `Email` as it did up until now."
```

File: Packs/MISP/ReleaseNotes/2_1_43.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### MISP v3
+
+Fixed an issue in `Base` pack (Version `1_33_52`) so now MISP will correctly input email addresses into context under `Accounts.Email` and not under `Email`.
```
Comment (ShahafBenYakir) [inline Packs/MISP/ReleaseNotes/2_1_43.md:6]:
```suggestion
Fixed an issue in `Base` pack (Version `1.33.52`) so now MISP will correctly input email addresses into context under `Accounts.Email` and not under `Email`.
```

File: Packs/MISP/ReleaseNotes/2_1_43.json (hunk 5 above)
Comment (ShahafBenYakir) [reply Packs/MISP/ReleaseNotes/2_1_43.json:3]: Can we mention the specific command/s that is impacted?

File: Packs/MISP/ReleaseNotes/2_1_43.md (hunk 6 above)
Comment (ShahafBenYakir) [reply Packs/MISP/ReleaseNotes/2_1_43.md:6]: Can we mention the specific command/s that is impacted?

**Review by ShahafBenYakir — COMMENTED** [review]
I am confused, why have we decided to added specific rn only to MISP and not all th packs?

**Review by ShahafBenYakir — APPROVED** [review]

**Review by MislavReversingLabs — APPROVED** [review]

**Review by MislavReversingLabs — APPROVED** [review]

**Pull request merged**